From be9c19ad0580899b14e27652b07203f141661a2c Mon Sep 17 00:00:00 2001
From: Nenad Ilic <nenadilic84@gmail.com>
Date: Tue, 25 Apr 2023 21:49:53 +0200
Subject: [PATCH] Add GGv2 FleetProvisioning

Upstream-Status: Inappropriate [oe-specific]
---
 bin/loader | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/loader b/bin/loader
index 5bcec52..8e86513 100644
--- a/bin/loader
+++ b/bin/loader
@@ -31,7 +31,7 @@ launch_kernel() {
   fi
 
   JVM_OPTIONS="$JVM_OPTIONS -Droot=$GG_ROOT"
-  OPTIONS="--setup-system-service false"
+  OPTIONS="--setup-system-service false --trusted-plugin $GG_ROOT/plugins/trusted/aws.greengrass.FleetProvisioningByClaim.jar"
   if [ ! -z "${CONFIG_FILE}" ]; then
     OPTIONS="$OPTIONS --config $CONFIG_FILE"
   fi
