Upstream-Status: Inappropriate [oe-specific]

Index: greengrass-bin/bin/loader
===================================================================
--- greengrass-bin.orig/bin/loader
+++ greengrass-bin/bin/loader
@@ -31,7 +31,7 @@ launch_kernel() {
   fi

   JVM_OPTIONS="$JVM_OPTIONS -Droot=$GG_ROOT"
-  OPTIONS="--setup-system-service false"
+  OPTIONS="--setup-system-service false --trusted-plugin $GG_ROOT/plugins/trusted/aws.greengrass.FleetProvisioningByClaim.jar"
   if [ ! -z "${CONFIG_FILE}" ]; then
     OPTIONS="$OPTIONS --config $CONFIG_FILE"
   fi
