From 4f85a8719ce9a21140c1fd89e6cf959747bc6d67 Mon Sep 17 00:00:00 2001
From: Rephael Congmon <rephcongmon@gmail.com>
Date: Fri, 5 Nov 2021 10:31:19 -0700
Subject: [PATCH] Added missing macro

---
 include/aws/mqtt/client.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/aws/mqtt/client.h b/include/aws/mqtt/client.h
index 06c8636..9263f26 100644
--- a/include/aws/mqtt/client.h
+++ b/include/aws/mqtt/client.h
@@ -418,6 +418,7 @@ int aws_mqtt_client_connection_connect(
  * \returns AWS_OP_SUCCESS if the connection has been successfully initiated,
  *              otherwise AWS_OP_ERR and aws_last_error() will be set.
  */
+AWS_MQTT_API
 int aws_mqtt_client_connection_reconnect(
     struct aws_mqtt_client_connection *connection,
     aws_mqtt_client_on_connection_complete_fn *on_connection_complete,
-- 
2.33.0

