From bb337b5d4228cc0e891f1b9320df28d0ebeaaf14 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Thu, 27 Nov 2025 12:49:26 +0000
Subject: [PATCH] aws-greengrass-component-sdk: use bindgen to correct create

Upstream-Status: Inappropriate [Yocto-specific]
---
 rust/Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index 0e8d846..5520625 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -9,6 +9,9 @@ readme = "../README.md"
 keywords = ["aws", "greengrass", "iot"]
 categories = ["api-bindings"]
 
+[lib]
+crate-type = ["cdylib", "rlib"]
+
 [build-dependencies]
 cc = "1"
 bindgen = "0.72"
