From 04a37b6672e84b12566d989cac0829c3ec634930 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 29 Sep 2025 12:51:03 +0000
Subject: [PATCH] This enable the tests even when crosscompiling.

Upstream-Status: Inappropriate [oe specific]
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 91b405b..9e39b3a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -222,7 +222,7 @@ install(FILES ${EXPORT_MODULES}
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/modules"
         COMPONENT Development)
 
-if (NOT CMAKE_CROSSCOMPILING AND NOT BYO_CRYPTO)
+if (NOT BYO_CRYPTO)
     include(CTest)
     if (BUILD_TESTING)
         add_subdirectory(bin/sha256_profile)
