From 28d2373d9033d969bc178c2c6d8626634fd65ebe Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Thu, 4 May 2023 12:31:54 +0000
Subject: [PATCH] aws-crt-python: upgrade 0.16.15 -> 0.16.16, readd shared

Upstream-Status: Denied [Desired default]
---
 setup.py | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/setup.py b/setup.py
index 32873b2..06ee926 100644
--- a/setup.py
+++ b/setup.py
@@ -330,16 +330,6 @@ def awscrt_ext():
         extra_link_args += ['-framework', 'Security']
 
     else:  # unix
-        # linker will prefer shared libraries over static if it can find both.
-        # force linker to choose static variant by using using
-        # "-l:libaws-c-common.a" syntax instead of just "-laws-c-common".
-        #
-        # This helps AWS developers creating Lambda applications from Brazil.
-        # In Brazil, both shared and static libs are available.
-        # But Lambda requires all shared libs to be explicitly packaged up.
-        # So it's simpler to link them in statically and have less runtime dependencies.
-        libraries = [':lib{}.a'.format(x) for x in libraries]
-
         # OpenBSD doesn't have librt; functions are found in libc instead.
         if not sys.platform.startswith('openbsd'):
             libraries += ['rt']
