From adcb173ccbb269624b493c9ae6e9e9ded86f6e50 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 18 Nov 2024 19:32:41 +0000
Subject: [PATCH] aws-iot-device-client: disable tests, as the new aws-c-io

Upstream-Status: Submitted [author]
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7f3a8c0..e7374fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -266,4 +266,4 @@ if (BUILD_TEST_DEPS)
             ${CMAKE_BINARY_DIR}/googletest-build)
 endif ()
 
-add_subdirectory(test)
+# add_subdirectory(test)
