Index: git/CMakeLists.txt
===================================================================
--- git.orig/CMakeLists.txt
+++ git/CMakeLists.txt
@@ -195,7 +195,7 @@ include_directories("${PROJECT_BINARY_DI
 if (MSVC)
     target_compile_options(${DC_PROJECT_NAME} PRIVATE /W4 /WX)
 else ()
-    target_compile_options(${DC_PROJECT_NAME} PRIVATE -Wall -Wno-long-long -pedantic -Werror)
+    target_compile_options(${DC_PROJECT_NAME} PRIVATE -Wall -Wno-long-long)
 endif ()
 
 set(DEP_DC_LIBS ${DEP_DC_LIBS} aws-crt-cpp aws-c-mqtt aws-c-auth aws-c-io aws-c-cal aws-c-http aws-c-compression aws-c-common aws-c-s3 aws-c-event-stream aws-checksums aws-c-sdkutils)
