From b38f182a9e22eefa67db7600198af49367810d9b Mon Sep 17 00:00:00 2001
From: rpcme <rich@richelberger.com>
Date: Sun, 16 Jan 2022 06:00:06 -0500
Subject: [PATCH] add support for amazon kvs producer sdk cpp

Upstream-Status: Submitted [author]

Index: amazon-kvs-producer-sdk-cpp-3.5.0/CMakeLists.txt
===================================================================
--- amazon-kvs-producer-sdk-cpp-3.5.0.orig/CMakeLists.txt
+++ amazon-kvs-producer-sdk-cpp-3.5.0/CMakeLists.txt
@@ -92,13 +92,15 @@ set(BUILD_COMMON_LWS
 set(BUILD_COMMON_CURL
     TRUE
     CACHE BOOL "Build ProducerC with CURL Support" FORCE)
+if(BUILD_DEPENDENCIES)
 set(DEPENDENCY_DOWNLOAD_PATH ${CMAKE_CURRENT_SOURCE_DIR}/dependency)
 if(NOT EXISTS ${DEPENDENCY_DOWNLOAD_PATH})
   file(MAKE_DIRECTORY ${DEPENDENCY_DOWNLOAD_PATH})
 endif()

 fetch_repo(kvscproducer)
-add_subdirectory(${DEPENDENCY_DOWNLOAD_PATH}/libkvscproducer/kvscproducer-src EXCLUDE_FROM_ALL)
+add_subdirectory(${DEPENDENCY_DOWNLOAD_PATH}/libkvscproducer/kvscproducer-src EXCLUDE_FROM_ALL)
+endif()
 ############# find dependent libraries ############

 find_package(Threads)
