From 29f76d9358e39561e35f12687efd7f96310c48ef Mon Sep 17 00:00:00 2001
From: rpcme <rich@richelberger.com>
Date: Sun, 16 Jan 2022 06:00:06 -0500
Subject: [PATCH] add support for amazon kvs producer sdk cpp

Upstream-Status: Submitted [author]

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec001a7..470e069 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,6 +94,7 @@ set(BUILD_COMMON_LWS
 set(BUILD_COMMON_CURL
     TRUE
     CACHE BOOL "Build ProducerC with CURL Support" FORCE)
+if(BUILD_DEPENDENCIES)
 set(DEPENDENCY_DOWNLOAD_PATH ${CMAKE_CURRENT_SOURCE_DIR}/dependency)
 if(NOT EXISTS ${DEPENDENCY_DOWNLOAD_PATH})
   file(MAKE_DIRECTORY ${DEPENDENCY_DOWNLOAD_PATH})
@@ -101,6 +102,8 @@ endif()
 fetch_repo(kvscproducer)
 add_subdirectory(${DEPENDENCY_DOWNLOAD_PATH}/libkvscproducer/kvscproducer-src EXCLUDE_FROM_ALL)

+endif()
+
 ############# find dependent libraries ############

 find_package(Threads)
