diff --git a/CMakeLists.txt b/CMakeLists.txt
index 60ecca1..3ce58be 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -202,8 +207,8 @@ if(BUILD_GSTREAMER_PLUGIN)
   add_library(gstkvssink MODULE ${GST_PLUGIN_SOURCE_FILES})
   target_link_libraries(gstkvssink PRIVATE ${GST_APP_LIBRARIES} KinesisVideoProducer)
 
-  add_executable(kvs_gstreamer_sample samples/kvs_gstreamer_sample.cpp)
-  target_link_libraries(kvs_gstreamer_sample ${GST_APP_LIBRARIES} KinesisVideoProducer)
+  #  add_executable(kvs_gstreamer_sample samples/kvs_gstreamer_sample.cpp)
+  #  target_link_libraries(kvs_gstreamer_sample ${GST_APP_LIBRARIES} KinesisVideoProducer)
 
   add_executable(kvs_gstreamer_multistream_sample samples/kvs_gstreamer_multistream_sample.cpp)
   target_link_libraries(kvs_gstreamer_multistream_sample ${GST_APP_LIBRARIES} KinesisVideoProducer)
