From 55a765ca7c0fdc46a0cfac8f7cbf10a89d42ded3 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Fri, 10 Mar 2023 12:46:05 +0000
Subject: [PATCH] aws-iot-securetunneling-localproxy: support any boost version

Upstream-Status: Denied [Any version that compile should be good, team does only support a specific version]
---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d54b8eb..34c897a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,8 +76,7 @@ endif(BUILD_TESTS)
 set(Boost_USE_STATIC_LIBS ON)
 set(Boost_USE_DEBUG_RUNTIME OFF)
 #set_property(GLOBAL PROPERTY Boost_USE_MULTITHREADED ON)
-set(BOOST_PKG_VERSION "1.81.0" CACHE STRING "")
-find_package(Boost ${BOOST_PKG_VERSION} REQUIRED COMPONENTS system log log_setup thread program_options date_time filesystem chrono)
+find_package(Boost REQUIRED COMPONENTS system log log_setup thread program_options date_time filesystem chrono)
 include_directories(${Boost_INCLUDE_DIRS})
 foreach(BOOST_LIB ${Boost_LIBRARIES})
     string(REPLACE ${CMAKE_SHARED_LIBRARY_SUFFIX} ${CMAKE_STATIC_LIBRARY_SUFFIX} BOOST_STATIC_LIB ${BOOST_LIB})
