From 0ac519e1cf096c70aa5deceff51c7f533a95eaf7 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 11 Mar 2024 10:35:55 +0000
Subject: [PATCH] aws-iot-securetunneling-localproxy: boost 1.84 format include

Upstream-Status: Submitted [author]

---
 src/config/ConfigFile.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/config/ConfigFile.cpp b/src/config/ConfigFile.cpp
index 105ca10..ce9f17b 100644
--- a/src/config/ConfigFile.cpp
+++ b/src/config/ConfigFile.cpp
@@ -19,6 +19,7 @@
 #include <boost/log/expressions.hpp>
 #include <boost/asio.hpp>
 #include <boost/filesystem.hpp>
+#include <boost/format.hpp>
 #include <boost/property_tree/ini_parser.hpp>
 #include <boost/algorithm/string.hpp>
 #include <boost/lexical_cast.hpp>
@@ -212,4 +213,4 @@ namespace aws { namespace iot { namespace securedtunneling { namespace config_fi
         boost::filesystem::path full_path(boost::filesystem::current_path());
         return (boost::format("%1%/config") % full_path.string()).str();
     }
-}}}}
\ No newline at end of file
+}}}}
