From 36fa8d748cf149a5db554cf18b4485b601d9dcc0 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 8 Jan 2024 15:07:56 +0000
Subject: [PATCH] aws-iot-securetunneling-localproxy: fix GCC 13 build

Upstream-Status: Pending -> https://github.com/aws-samples/aws-iot-securetunneling-localproxy/pull/136
---
 src/Url.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Url.cpp b/src/Url.cpp
index 3323706..7ff2b23 100644
--- a/src/Url.cpp
+++ b/src/Url.cpp
@@ -107,4 +107,4 @@ string aws::iot::securedtunneling::url::url_decode(const string &url_s) {
         }
         return out;
     }
-}
\ No newline at end of file
+}
