From 2e1f47ca467a3a5a1e1a56879b784bc33599eb76 Mon Sep 17 00:00:00 2001
From: tro <throos@amazon.de>
Date: Tue, 30 May 2023 12:10:36 +0200
Subject: [PATCH] ProxySettings.h: add #include <cstdint>

Upstream-Status: Pending -> https://github.com/aws-samples/aws-iot-securetunneling-localproxy/pull/136
---
 src/ProxySettings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ProxySettings.h b/src/ProxySettings.h
index 9dc4e10..de3098a 100644
--- a/src/ProxySettings.h
+++ b/src/ProxySettings.h
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 #pragma once
 
+#include <cstdint>
 #include <boost/property_tree/ptree.hpp>
 namespace aws { namespace iot { namespace securedtunneling { namespace settings {
     using boost::property_tree::ptree;
