From be61db954f8819dd7252b86f8aa5716da0d4d0b2 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Tue, 9 Jan 2024 14:34:32 +0000
Subject: [PATCH] aws-iot-securetunneling-localproxy: remove setting of

Upstream-Status: Submitted [author]

---
 CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 34c897a..972e5a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,3 @@
-cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
-
 set(AWS_TUNNEL_LOCAL_PROXY_TARGET_NAME localproxy)
 set(AWS_TUNNEL_LOCAL_PROXY_LIB_NAME lproxy)
 project(${AWS_TUNNEL_LOCAL_PROXY_TARGET_NAME} CXX)
@@ -21,9 +19,7 @@ endif ()
 ########################################
 # Section : Common Build setttings #
 ########################################
-# Set required compiler standard to standard c++11. Disable extensions.
-set(CMAKE_CXX_STANDARD 14) # C++14
-set(CMAKE_CXX_STANDARD_REQUIRED ON) #...is required...
+# Disable extensions.
 set(CMAKE_CXX_EXTENSIONS OFF) #...without compiler extensions like gnu++11

 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
