Upstream-Status: Inappropriate [oe specific]

Index: git/modules/ggl-http/src/gghttp.c
===================================================================
--- git.orig/modules/ggl-http/src/gghttp.c
+++ git/modules/ggl-http/src/gghttp.c
@@ -7,7 +7,6 @@
 #include "ggl/error.h"
 #include "ggl/http.h"
 #include <assert.h>
-#include <curl/curl.h>
 #include <ggl/buffer.h>
 #include <ggl/log.h>
 #include <ggl/vector.h>
@@ -160,17 +159,6 @@ GglError sigv4_download(
         *http_response_code = 400;
     }

-    struct curl_header *type = NULL;
-    curl_easy_header(
-        curl_data.curl, "Content-Type", 0, CURLH_HEADER, -1, &type
-    );
-
-    if (type) {
-        GGL_LOGT("Response header content: %s", type->value);
-    } else {
-        GGL_LOGT("Response header content: is empty");
-    }
-
     gghttplib_destroy_curl(&curl_data);

     return error;
