Upstream-Status: Submitted [https://github.com/aws/aws-iot-device-sdk-cpp-v2/issues/776]

Index: git/CMakeLists.txt
===================================================================
--- git.orig/CMakeLists.txt
+++ git/CMakeLists.txt
@@ -37,10 +37,7 @@ if (${CMAKE_INSTALL_LIBDIR} STREQUAL "li
 endif()

 # This is required in order to append /lib/cmake to each element in CMAKE_PREFIX_PATH
-set(AWS_MODULE_DIR "/${CMAKE_INSTALL_LIBDIR}/cmake")
-string(REPLACE ";" "${AWS_MODULE_DIR};" AWS_MODULE_PATH "${CMAKE_PREFIX_PATH}${AWS_MODULE_DIR}")
-# Append that generated list to the module search path
-list(APPEND CMAKE_MODULE_PATH ${AWS_MODULE_PATH})
+find_package(aws-c-common REQUIRED)

 if (NOT CMAKE_BUILD_TYPE)
     if (NOT WIN32)
