<<<<<<< HEAD
From 980cddcfefd4494b11f6893653687faf8fa3c84b Mon Sep 17 00:00:00 2001
=======
From 319442d6c208826e76d39062f6aff85c3b2f005e Mon Sep 17 00:00:00 2001
>>>>>>> 0c5312513 (aws-c-io: upgrade 0.27.2 -> 0.27.3)
From: Thomas Roos <throos@amazon.de>
Date: Mon, 29 Sep 2025 12:51:03 +0000
Subject: [PATCH] This enable the tests even when crosscompiling.

Upstream-Status: Inappropriate [oe specific]
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
<<<<<<< HEAD
index 191d1b6..ae2b846 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -255,8 +255,6 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
=======
index dd5eeb8..b48fb29 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -261,8 +261,6 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
>>>>>>> 0c5312513 (aws-c-io: upgrade 0.27.2 -> 0.27.3)
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/"
         COMPONENT Development)
 
-if (NOT CMAKE_CROSSCOMPILING)
     if (BUILD_TESTING)
        add_subdirectory(tests)
     endif()
-endif()
