From ea081726e1ac1df3039ee64855d79f368e8de890 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 29 Sep 2025 12:51:03 +0000
Subject: [PATCH] This enable the tests even when crosscompiling.

Upstream-Status: Inappropriate [oe specific]
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0eafb22..ca21dc2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -91,9 +91,7 @@ install(FILES ${EXPORT_MODULES}
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/modules"
         COMPONENT Development)
 
-if (NOT CMAKE_CROSSCOMPILING)
     include(CTest)
     if (BUILD_TESTING)
         add_subdirectory(tests)
     endif()
-endif()
