From 75e42cb04a6798322d7601004254ae2b42271a10 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Tue, 9 Jan 2024 15:01:35 +0000
Subject: [PATCH] aws-iot-fleetwise-edge: remove setting of cxx-standard

Upstream-Status: Submitted [author]

---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1f6c041..8b74eae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,11 +4,6 @@ cmake_minimum_required(VERSION 3.10.2)

 project(iotfleetwise VERSION 1.1.1)

-# FWE uses C++14 for compatibility reasons with Automotive middlewares (Adaptive AUTOSAR, ROS2)
-# Note: When built with FWE_FEATURE_ROS2, colcon will override these settings
-set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED True)
-
 # Print out the compile commands which is useful for IDEs
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
