Index: git/CMakeLists.txt
===================================================================
--- git.orig/CMakeLists.txt
+++ git/CMakeLists.txt
@@ -4,11 +4,6 @@ cmake_minimum_required(VERSION 3.10.2)
 
 project(iotfleetwise VERSION 1.1.0)
 
-# FWE uses C++14 for compatibility reasons with Automotive middlewares (Adaptive AUTOSAR, ROS2)
-# Note: When built with FWE_FEATURE_ROS2, colcon will override these settings
-set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED True)
-
 # Print out the compile commands which is useful for IDEs
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
