From e3a8b0ccb2249abed1322dba8fdc241498d28bbc Mon Sep 17 00:00:00 2001
From: nateglims <glimsdal@amazon.com>
Date: Fri, 27 May 2022 17:52:02 +0000
Subject: [PATCH] Remove abort panic from the workspace

---
 Cargo.toml | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 8c3056c9..84e8cc3e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -2,11 +2,7 @@
 members = ["src/firecracker", "src/jailer", "src/seccompiler", "src/rebase-snap"]
 default-members = ["src/firecracker"]
 
-[profile.dev]
-panic = "abort"
-
 [profile.release]
-panic = "abort"
 lto = true
 
 [patch.crates-io]
