diff --git a/CMakeLists.txt b/CMakeLists.txt
index c7993b9..f422abf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -286,24 +286,24 @@ add_custom_target(demo DEPENDS ${DEMO_EXECS})
 
 # Tests
 if(NOT(AAR_BUILD))
-  include(cmake/googletest.cmake)
-  if (WIN32)
-    include_directories(${DLFCN_WIN32}/src)
-    add_subdirectory(${DLFCN_WIN32})
-  endif()
-
-  fetch_googletest(
-    ${PROJECT_SOURCE_DIR}/cmake
-    ${PROJECT_BINARY_DIR}/googletest
-  )
-
-  enable_testing()
-  file(GLOB TEST_SRCS tests/cpp/*.cc)
-
-  if(WITH_HEXAGON)
-    file(GLOB HEXAGON_TEST_SRCS tests/cpp/dlr_hexagon/*.cc)
-    list(APPEND TEST_SRCS ${HEXAGON_TEST_SRCS})
-  endif()
+  include(/worktree/poky/build/tmp/work/core2-64-poky-linux/googletest/1.10.0-r0/sysroot-destdir/usr/lib/cmake/GTest/GTestConfig.cmake)
+# if (WIN32)
+#    include_directories(${DLFCN_WIN32}/src)
+#    add_subdirectory(${DLFCN_WIN32})
+#  endif()
+
+#  fetch_googletest(
+#    ${PROJECT_SOURCE_DIR}/cmake
+#    ${PROJECT_BINARY_DIR}/googletest
+#  )
+
+#  enable_testing()
+#  file(GLOB TEST_SRCS tests/cpp/*.cc)
+
+#  if(WITH_HEXAGON)
+#    file(GLOB HEXAGON_TEST_SRCS tests/cpp/dlr_hexagon/*.cc)
+#    list(APPEND TEST_SRCS ${HEXAGON_TEST_SRCS})
+#  endif()
   foreach(__srcpath ${TEST_SRCS})
     get_filename_component(__srcname ${__srcpath} NAME)
     string(REPLACE ".cc" "" __execname ${__srcname})
