From 489a2e58d994845dd09311d95604264a9c30982b Mon Sep 17 00:00:00 2001
From: rpcme <rich@richelberger.com>
Date: Wed, 2 Jun 2021 18:46:28 -0400
Subject: [PATCH] overcome build break due to GCC 11.1 add that exposed

---
 CMakeLists.txt | 20 ++++++++++++++------
 1 file changed, 14 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3a0d3ea1..12d1c6b6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -347,14 +347,22 @@ file(GLOB S2N_SRC
 )
 
 add_library(${PROJECT_NAME} ${S2N_HEADERS} ${S2N_SRC})
-set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE C)
 
-set(CMAKE_C_FLAGS_DEBUGOPT "")
 
-target_compile_options(${PROJECT_NAME} PRIVATE -pedantic -std=gnu99 -Wall -Wimplicit -Wunused -Wcomment -Wchar-subscripts
-        -Wuninitialized -Wshadow -Wcast-align -Wwrite-strings -Wno-deprecated-declarations -Wno-unknown-pragmas -Wformat-security
-        -Wno-missing-braces -Wa,--noexecstack
-)
+set_target_properties(${PROJECT_NAME}
+                      PROPERTIES LINKER_LANGUAGE C
+                                 C_STANDARD 99 )
+
+target_compile_options(${PROJECT_NAME} PRIVATE -Wa,--noexecstack -DS2N_EXPORTS)
+
+#set extra warning flags for debug build
+if (CMAKE_BUILD_TYPE STREQUAL "" OR CMAKE_BUILD_TYPE MATCHES Debug)
+    target_compile_options(${PROJECT_NAME} PRIVATE -pedantic -Wall -Werror -Wimplicit -Wunused -Wcomment )
+    target_compile_options(${PROJECT_NAME} PRIVATE -Wchar-subscripts -Wuninitialized -Wshadow )
+    target_compile_options(${PROJECT_NAME} PRIVATE -Wcast-qual -Wcast-align -Wwrite-strings )
+    target_compile_options(${PROJECT_NAME} PRIVATE -Wno-deprecated-declarations -Wno-unknown-pragmas )
+    target_compile_options(${PROJECT_NAME} PRIVATE -Wformat-security -Wno-missing-braces )
+endif ()
 
 if (UNSAFE_TREAT_WARNINGS_AS_ERRORS)
     target_compile_options(${PROJECT_NAME} PRIVATE -Werror )
