From 8a2fb9b14ea7346f61115b0a95286259a6973132 Mon Sep 17 00:00:00 2001
From: Samuel Chiang <sachiang@amazon.com>
Date: Thu, 16 Mar 2023 12:19:28 -0700
Subject: [PATCH] Update analytics script to handle new tool name from #865
 (#873)

---
 tests/ci/run_analytics.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/ci/run_analytics.sh b/tests/ci/run_analytics.sh
index 1bf9a2878..d4470e6fa 100755
--- a/tests/ci/run_analytics.sh
+++ b/tests/ci/run_analytics.sh
@@ -84,7 +84,7 @@ function run_build_and_collect_metrics {
 
   libcrypto_size=$(size "${BUILD_ROOT}/crypto/libcrypto.${lib_extension}")
   libssl_size=$(size "${BUILD_ROOT}/ssl/libssl.${lib_extension}")
-  tool_size=$(size "${BUILD_ROOT}/tool/bssl")
+  tool_size=$(size "${BUILD_ROOT}/tool/awslc")
   put_metric --metric-name LibCryptoSize --value "$libcrypto_size" --unit Bytes --dimensions "${size_common_dimensions}"
   put_metric --metric-name LibSSLSize --value "$libssl_size" --unit Bytes --dimensions "${size_common_dimensions}"
   put_metric --metric-name ToolSize --value "$tool_size" --unit Bytes --dimensions "${size_common_dimensions}"
@@ -117,8 +117,8 @@ no_assembly=OFF
 fips=ON
 run_build_and_collect_metrics
 
-# The static FIPS build does not work on ARM with GCC, fix tracked in CryptoAlg-1399
-if [[ ("$(uname -s)" == 'Linux'*) && (("$(uname -p)" == 'x86_64'*) || (("$(uname -p)" == 'aarch64') && ("$CC" == 'clang'*))) ]]; then
+# The static FIPS build only works on Linux platforms.
+if [[ ("$(uname -s)" == 'Linux'*) && (("$(uname -p)" == 'x86_64'*) || ("$(uname -p)" == 'aarch64'*)) ]]; then
   shared_library=OFF
   run_build_and_collect_metrics
 fi
