From 823ae7fa8764914f23a359f034d81642f1ce7c74 Mon Sep 17 00:00:00 2001
From: Justin W Smith <103147162+justsmth@users.noreply.github.com>
Date: Fri, 21 Apr 2023 10:31:32 -0400
Subject: [PATCH] Clarify EC_POINT_point2bn docs (#966)

---
 include/openssl/ec.h | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/include/openssl/ec.h b/include/openssl/ec.h
index fe1e07418..6192a4b59 100644
--- a/include/openssl/ec.h
+++ b/include/openssl/ec.h
@@ -372,9 +372,11 @@ OPENSSL_EXPORT int EC_GROUP_set_generator(EC_GROUP *group,
                                           const BIGNUM *cofactor);
 
 
-// EC_POINT_point2bn converts an |EC_POINT| to |BIGNUM|. On success, returns the
-// BIGNUM pointer supplied, |ret|. Otherwise, it returns NULL on error. The
-// |ctx| argument may be used if not NULL.
+// EC_POINT_point2bn converts an |EC_POINT| to a |BIGNUM| by serializing the
+// point into the X9.62 form given by |form| then interpretting it as a BIGNUM.
+// On success, it returns the BIGNUM pointer supplied or, if |ret| is NULL,
+// allocates and returns a fresh |BIGNUM|. On error, it returns NULL. The |ctx|
+// argument may be used if not NULL.
 AWS_LC_DEPRECATED OPENSSL_EXPORT BIGNUM *EC_POINT_point2bn(
     const EC_GROUP *group, const EC_POINT *point, point_conversion_form_t form,
     BIGNUM *ret, BN_CTX *ctx);
