From eeab3e8080218282a86210877feaafc77daf9b7c Mon Sep 17 00:00:00 2001
From: Subhajit Paul <subhajit_paul@ti.com>
Date: Sat, 29 Aug 2020 21:35:27 +0530
Subject: [PATCH] [maybe-upstream] add condition for googletest build

Signed-off-by: Subhajit Paul <subhajit_paul@ti.com>
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4173f9b..3d853b7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,6 +5,7 @@ include(3rdparty/tvm/cmake/util/FindCUDA.cmake)
 # Option for Android on Arm --- has to come before project() function
 option(ANDROID_BUILD "Build for Android target" OFF)
 option(AAR_BUILD "Build Android Archive (AAR)" OFF)
+option(USE_TESTS "Build tests" ON)
 
 if(ANDROID_BUILD)
     set(ANDROID_SYSROOT "${NDK_ROOT}/sysroot")
@@ -334,7 +335,7 @@ endforeach()
 add_custom_target(demo DEPENDS ${DEMO_EXECS})
 
 # Tests
-if(NOT(AAR_BUILD))
+if(NOT(AAR_BUILD) AND USE_TESTS)
   include(cmake/googletest.cmake)
   fetch_googletest(
     ${PROJECT_SOURCE_DIR}/cmake
-- 
2.17.1

