From 37ddd83833678643b899f5789e15038cfd4affb9 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 29 Sep 2025 12:51:03 +0000
Subject: [PATCH] This enable the tests even when crosscompiling.

Upstream-Status: Inappropriate [oe specific]
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d003cab..e10bf79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -426,7 +426,6 @@ install(FILES "${GENERATED_ROOT_DIR}/${PROJECT_NAME}-config-version.cmake"
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/"
     COMPONENT Development)
 
-if(NOT CMAKE_CROSSCOMPILING)
     if(BUILD_TESTING)
         add_subdirectory(tests)
 
@@ -436,4 +435,3 @@ if(NOT CMAKE_CROSSCOMPILING)
             add_subdirectory(bin/mqtt5_canary)
         endif()
     endif()
-endif()
