From 3c9c3f4b5b7137f1ddbf6cbc9e4dfe861cd2c823 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Mon, 29 Sep 2025 12:51:03 +0000
Subject: [PATCH] This enable the tests even when crosscompiling.

Upstream-Status: Inappropriate [oe specific]
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1f8d466..fdfbc32 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -446,7 +446,6 @@ install(FILES "${GENERATED_ROOT_DIR}/${PROJECT_NAME}-config-version.cmake"
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/"
     COMPONENT Development)
 
-if(NOT CMAKE_CROSSCOMPILING)
     if(BUILD_TESTING)
         add_subdirectory(tests)
 
@@ -456,4 +455,3 @@ if(NOT CMAKE_CROSSCOMPILING)
             add_subdirectory(bin/mqtt5_canary)
         endif()
     endif()
-endif()
