diff --git a/CMakeLists.txt b/CMakeLists.txt
index ee5169f..ad88097 100644
--- a/amazon-kvs-producer-sdk-c/CMakeLists.txt
+++ b/amazon-kvs-producer-sdk-c/CMakeLists.txt
@@ -95,6 +95,7 @@ endif()
 
 ############# fetch repos that we need do add_subdirectory ############
 
+if(BUILD_DEPENDENCIES)
 # repos that we will build using add_subdirectory will be stored in this path
 set(DEPENDENCY_DOWNLOAD_PATH ${CMAKE_CURRENT_SOURCE_DIR}/dependency)
 set(BUILD_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
@@ -102,7 +103,7 @@ fetch_repo(kvspic ${BUILD_ARGS})
 add_subdirectory("${DEPENDENCY_DOWNLOAD_PATH}/libkvspic/kvspic-src")
 file(GLOB PIC_HEADERS "${pic_project_SOURCE_DIR}/src/*/include")
 include_directories("${PIC_HEADERS}")
-
+endif()
 ############# fetch repos that we need do add_subdirectory done ############
 
 ############# find dependent libraries ############
