From 9d92debf2608d4937919d43cf1dcaba51333b535 Mon Sep 17 00:00:00 2001
From: OpenEmbedded <oe.patch@oe>
Date: Fri, 1 Sep 2023 00:34:07 +0000
Subject: [PATCH] amazon-kvs-webrtc-sdk: upgrade git -> new

---
 .../libkvsCommonLws-CMakeLists.txt            | 36 ++++++++++---------
 1 file changed, 19 insertions(+), 17 deletions(-)

diff --git a/CMake/Dependencies/libkvsCommonLws-CMakeLists.txt b/CMake/Dependencies/libkvsCommonLws-CMakeLists.txt
index b68c3c638..c6bf682fc 100644
--- a/CMake/Dependencies/libkvsCommonLws-CMakeLists.txt
+++ b/CMake/Dependencies/libkvsCommonLws-CMakeLists.txt
@@ -4,20 +4,22 @@ project(libkvsCommonLws-download NONE)
 
 include(ExternalProject)
 
-ExternalProject_Add(libkvsCommonLws-download
-    GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           7d1b76f53680c1e23afb6f35f0cca97ccdb35e3f
-    PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/build
-    CMAKE_ARGS        
-      -DCMAKE_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
-      -DBUILD_COMMON_LWS=ON
-      -DBUILD_COMMON_CURL=OFF
-      -DBUILD_DEPENDENCIES=FALSE
-      -DOPEN_SRC_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
-      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-      -DUSE_OPENSSL=${USE_OPENSSL}
-      -DUSE_MBEDTLS=${USE_MBEDTLS}
-      -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
-    BUILD_ALWAYS      TRUE
-    TEST_COMMAND      ""
-)
+if(BUILD_DEPENDENCIES)
+ ExternalProject_Add(libkvsCommonLws-download
+     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
+     GIT_TAG           7d1b76f53680c1e23afb6f35f0cca97ccdb35e3f
+     PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/build
+     CMAKE_ARGS        
+       -DCMAKE_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
+       -DBUILD_COMMON_LWS=ON
+       -DBUILD_COMMON_CURL=OFF
+       -DBUILD_DEPENDENCIES=FALSE
+       -DOPEN_SRC_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
+       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+       -DUSE_OPENSSL=${USE_OPENSSL}
+       -DUSE_MBEDTLS=${USE_MBEDTLS}
+       -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+     BUILD_ALWAYS      TRUE
+     TEST_COMMAND      ""
+ )
+endif()
